geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/kafka/commands/RemoveSchemaCommand.scala

```diff
@@ -0,0 +1,87 @@
+/***********************************************************************
+* Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Apache License, Version 2.0
+* which accompanies this distribution and is available at
+* http://www.opensource.org/licenses/apache2.0.php.
+*************************************************************************/
+
+package org.locationtech.geomesa.tools.kafka.commands
+
+import com.beust.jcommander.{JCommander, Parameters}
+import com.typesafe.scalalogging.LazyLogging
+import org.locationtech.geomesa.tools.common.{OptionalFeatureTypeNameParam, OptionalForceParam, OptionalPatternParam, PromptConfirm}
+import org.locationtech.geomesa.tools.kafka.SimpleProducerKDSConnectionParams
+import org.locationtech.geomesa.tools.kafka.commands.RemoveSchemaCommand.RemoveSchemaParams
+
+import scala.util.{Failure, Success, Try}
+
+class RemoveSchemaCommand(parent: JCommander) extends CommandWithKDS(parent) with LazyLogging {
+  override val command = "removeschema"
+  override val params = new RemoveSchemaParams
+
+  override def execute() = {
+    if (Option(params.pattern).isEmpty && Option(params.featureName).isEmpty) {
+      throw new IllegalArgumentException("Please provide either featureName or pattern to removing.")
+    }
+
+    val typeNamesToRemove = getTypeNamesFromParams()
+    validate(typeNamesToRemove) match {
+      case Success(_) =>
+        if (params.force || promptConfirm(typeNamesToRemove)) {
+          removeAll(typeNamesToRemove)
+        } else {
+          logger.info(s"Cancelled schema removal.")
+        }
+      case Failure(ex) =>
+        println(s"Feature validation failed on error: ${ex.getMessage}.")
```

Comment:
FYI we tweaked the println vs log messages so that we could have things to go stdout in a standard way and errors to go elsewhere if people wanted to pipe the results into something. we log to stderr by default so thats why we differentiate from std out.